src/main/resources/logback.xml

```diff
@@ -0,0 +1,52 @@
+<configuration>
+
+    <appender name="STDOUT" class="ch.qos.logback.core.ConsoleAppender">
+        <encoder>
+            <pattern>%d{HH:mm:ss.SSS} [%thread] %-5level %logger{36} - %msg%n</pattern>
+        </encoder>
+    </appender>
+
+    <appender name="AllAppender" class="ch.qos.logback.core.FileAppender">
+        <file>log/all.log</file>
+        <append>false</append>
+        <immediateFlush>true</immediateFlush>
+        <encoder>
+            <pattern>%d{HH:mm:ss.SSS} [%thread] %-5level %logger{36} - %msg%n</pattern>
+        </encoder>
+    </appender>
+
+    <appender name="ModelAppender" class="ch.qos.logback.core.FileAppender">
+        <file>log/model.log</file>
+        <append>true</append>
+        <immediateFlush>true</immediateFlush>
+        <encoder>
+            <pattern>%d{HH:mm:ss.SSS} [%thread] %-5level %logger{36} - %msg%n</pattern>
+        </encoder>
+    </appender>
+
+    <appender name="EvaluationAppender" class="ch.qos.logback.core.FileAppender">
+        <file>log/evaluator.log</file>
+        <append>true</append>
+        <immediateFlush>true</immediateFlush>
+        <encoder>
+            <pattern>%d{HH:mm:ss.SSS} [%thread] %-5level %logger{36} - %msg%n</pattern>
+        </encoder>
+    </appender>
+
+    <root level="INFO">
```

Comment:
why info

Reply:
Using info means that you are interested in three levels of logging (info, warn, error), I don't think trace and debug are important for us.

Reply:
maybe debug will be important as we can make spark logs or anything that have a lot of logs just for debugging as debugs and if we want to not see all of them we can then change this to be info

Reply:
I think we should keep it as it is. Debug is more used when you implement your own staff just it check it works correctly and makes it easier to find any bug if one exists. All the useful logs that we will need from spark will have the level info or above.